modules/persistent-storage-csi-smb-cifs-provision-static.adoc

```diff
@@ -0,0 +1,187 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-smb-cifs-provision-static_{context}"]
+= Static provisioning
+
+You can use static provisioning to create a persistent volume (PV) and persistent volume claim (PVC) to consume existing Server Message Block protocol (SMB) shares:
+
+.Prerequisites
+* Access to the {product-title} web console.
+* {FeatureName} CSI Driver Operator and driver installed.
+
+.Procedure
+
+To set up static provisioning:
+
+. Create a PV by running the following command with the following example YAML file:
```

Comment:
Do we want to include the secret creation again or do we assume it's clear from the last step? Knowing that it's not necessary the same secret as dynamic provisioning.

Reply:
I think that secret creation should be in the procedure, so I've added it.